### ISSUE TITLE
Year 2023

### ISSUE THREAD

**Issue by HariboInc** [nohello-net/site#147]
The year has changed haha

**Comment by CruseCtrl** [comment]
Judging by [this line](https://github.com/nohello-net/site/blob/449b1f4d776bb9c1f5c887f3cd15b7fb78183411/src/util/i18n.ts#L28), I think all it needs is a re-deployment

```ts
obj.currentYear = new Date().getFullYear().toString(10);
```

But it would be nice if it always used the current year regardless of when it was last deployed

**Comment by igormcsouza** [comment]
So, it means that function is only called on the deployment? I guess is simple to solve that.. is that any PR for this?

**Comment by CruseCtrl** [comment]
@igormcsouza yes, I think the function is only called on deployment. [Doesn't look like there are any PRs yet](https://github.com/nohello-net/site/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+year), feel free to raise one :)

**Comment by p-hash** [comment]
It's 2024, guys

**Comment by HariboInc** [comment]
> It's 2024, guys

You're right. Now it's 2024 already, sad. Would've seen it updated already but the site says enough already :D
